packages/ogmios/src/CardanoNode/OgmiosCardanoNode.ts

```diff
@@ -93,4 +108,25 @@ export class OgmiosCardanoNode implements CardanoNode {
       throw CardanoNodeUtil.asCardanoNodeError(error) || new CardanoNodeErrors.UnknownCardanoNodeError(error);
     }
   }
+
+  async healthCheck() {
```

Comment:
Maybe also add the return type? Not all the methods have it, but I think they should (even if CardanoNode has the return type)

Reply:
753ce1beadd3e717fa08387b303b9209f456c66f